test/grpc/test_csi.js

```diff
@@ -427,7 +444,17 @@ function csiProtocolTest (protoname, shareType, timeoutMillis) {
         client.nodeStageVolume(getDefaultArgs(), (err) => {
           if (err) return done(err);
           assert.equal(getFsType(mountTarget), 'xfs');
-          done();
+          // Other protocols than NVMF do not honor ioTimeout setting
+          if (protoName !== 'NVMF') return done();
+          let major = getFsDevice(mountTarget).match(/nvme(\d+)n1/)[1];
+          fs.readFile(
+            `/sys/class/nvme/nvme${major}/nvme${major}c1n1/queue/io_timeout`,
```

Comment:
This won't work where there is e.g. `/sys/class/nvme2/nvme0c2n1` for the case of connecting to 3 paths with the same nexus/subnqn. I think it should be `/sys/class/nvme/nvme{major}/nvme*n1/...` like in the CSI plugin itself.

Reply:
will fix it. thanks 👍